node/impl/full/eth.go

```diff
@@ -252,7 +252,7 @@ func (a *EthModule) parseBlkParam(ctx context.Context, blkParam string) (tipset
 		if err != nil {
 			return nil, fmt.Errorf("cannot parse block number: %v", err)
 		}
-		ts, err := a.Chain.GetTipsetByHeight(ctx, abi.ChainEpoch(num), nil, false)
+		ts, err := a.Chain.GetTipsetByHeight(ctx, abi.ChainEpoch(num), nil, true)
```

Comment:
Can you revert this change, since it's already in https://github.com/filecoin-project/lotus/pull/10427?

Reply:
The e2e test would fail if this line is not included. I can rebase if #10427 is merged.

Reply:
Gotcha, thanks.